src/StanfordSentimentTreebank.jl

```diff
@@ -0,0 +1,30 @@
+"""StanfordSentimentTreebank Type
+having dict and senti fields for
+phrases and corresponding sentiments respectively"""
+struct StanfordSentimentTreebank
+    filepath :: String
+end
+
+function StanfordSentimentTreebank()
+    path = joinpath(datadep"Stanford Sentiment Treebank", "sa_dataset.txt")
+    StanfordSentimentTreebank(path)
+end
+
+MultiResolutionIterators.levelname_map(::Type{StanfordSentimentTreebank}) = [
+    :documents => 1,
+    :sentences => 2,
+    :words => 3, :tokens => 3,
+    :characters => 4
+]
+
+function load(dataset::StanfordSentimentTreebank)
+    Channel(ctype=@NestedVector(Any, 1), csize=4) do docs
```

Comment:
This line is wrong.
Saying `@NestedVector(Any, 1)` is exactly the same as sayin
`Vector{Any}`

We want to property type the returned objects.

I would also like to know your rational behind  `csize=4`

Reply:
An alternative is below if it would not be too much data would be to not bother with the Channels.
They are only there to stop things that might otherwise use all the RAM,
so it this file is not to big then it is not needed.

But also there are a number of other problems with the above code.
Including the use of `file` for two different variables. and not using `eachline`
Note the other changes in the code below


```
function load(dataset::StanfordSentimentTreebank)
    map(eachline(dataset.filepath)) do line
        doc_raw, sentiment_raw = split(line, '|')
        sents = intern.(tokenize.(split_sentences(doc_raw)))
        sentiment = parse(Float64, sentiment_raw)
        put!(docs, [sents, sentiment])  # TODO: rethink this return type.
    end
end
```

Reply:
I thought of such return type so that sentences and their sentiments can be taken together. If you have any other suggestion please let me know

Reply:
> 
> 
> This line is wrong.
> Saying `@NestedVector(Any, 1)` is exactly the same as sayin
> `Vector{Any}`
> 
> We want to properly declare the type the returned objects.
> 
> I would also like to know your rational behind `csize=4`

Sorry for that I think I misunderstood `csize` thing